lib/tests/test_pragma.c

```diff
@@ -28,9 +28,9 @@
 
 Test(pragma_parser, process_valid_major_minor)
 {
-  const guint version = process_version_string("3.28");
+  const guint version = process_version_string("3.27");
 
-  cr_assert_eq(0x031c, version);
+  cr_assert_eq(0x031b, version);
 }
```

Comment:
this should not need to be changed every time the version number is updated, so we can leave it at whatever version, without changing it every time we do a release.